src/tracer.js

```diff
@@ -147,26 +147,22 @@ export default class Tracer {
      * within `carrier`. The expected type of `carrier` depends on the value of
      * `format.
      *
-     * OpenTracing defines a common set of `format` values (see FORMAT_TEXT_MAP
-     * and FORMAT_BINARY), and each has an expected carrier type.
+     * OpenTracing defines a common set of `format` values (see
+     * FORMAT_TEXT_MAP, FORMAT_HTTP_HEADERS, and FORMAT_BINARY), and each has
+     * an expected carrier type.
      *
      * Consider this pseudocode example:
      *
      *     var clientSpan = ...;
      *     ...
      *     // Inject clientSpan into a text carrier.
-     *     var textCarrier = {};
-     *     Tracer.inject(clientSpan.context(), Tracer.FORMAT_TEXT_MAP, textCarrier);
+     *     var headersCarrier = {};
+     *     Tracer.inject(clientSpan.context(), Tracer.FORMAT_HTTP_HEADERS, headersCarrier);
      *     // Incorporate the textCarrier into the outbound HTTP request header
      *     // map.
-     *     outboundHTTPReq.headers.extend(textCarrier);
+     *     outboundHTTPReq.headers.extend(headersCarrier);
```

Comment:
My comment is a little off-topic as this is both not new to the PR *and* the code comment is explicitly labeled as pseudo-code, but the docs here are a little confusing since `extend` is not a function.  Perhaps the comment could use this:

```js
Object.assign(outboundRequest.headers, headersCarrier);
```

`Object.assign` is at least an ES6 method and supported natively by Node. (Unfortunately, there's no pre-ES6 one-liner for merging associative arrays in JS that I'm aware of that doesn't involve an external lib).